File: csharp/raindrops/Raindrops.cs
public static class Raindrops
{
    public static string Convert(int number)
    {
        string sounds = "";
        if (number % 3 == 0)
        {
            sounds += "Pling";            
        }
        if (number % 5 == 0)
        {
            sounds += "Plang";            
        }
        if (number % 7 == 0)
        {
            sounds += "Plong";
        }
        return sounds.Length > 0 ? sounds : number.ToString();
    }
}

File: javascript/raindrops/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (num) => {
  let ret = ''
  let clean = true
  
  // is divisible by 3, add "Pling" to the result.
  if (num % 3 === 0)
  {
    ret += 'Pling'
    clean = false
  }

  // is divisible by 5, add "Plang" to the result.
  if (num % 5 === 0)
  {
    ret += 'Plang'
    clean = false
  }

  // is divisible by 7, add "Plong" to the result.
  if (num % 7 === 0)
  {
    ret += 'Plong'
    clean = false
  }

  // is not divisible by 3, 5, or 7, the result should be the number as a string.
  if (clean)
    ret += String(num)

  return ret
};
